db/db.go

```diff
@@ -17,22 +17,21 @@ import (
 
 	"roci.dev/diff-server/kv"
 	jsnoms "roci.dev/diff-server/util/noms/json"
-	"roci.dev/diff-server/util/time"
 )
 
 const (
-	LOCAL_DATASET  = "local"
+	MASTER_DATASET = "master"
```

Comment:
[minifix] We don't use `REMOTE` anymore and it can go away.

Reply:
yes, on my list. went to do it in this pr but doing so required changes to the log command line command and i decided to get into it later. stay tuned.